pysqldb3/data_io.py

```diff
@@ -94,6 +94,94 @@ def pg_to_sql(pg, ms, org_table, LDAP=False, spatial=True, org_schema=None, dest
+    :param ms: DbConnect instance connecting to SQL Server destination database
+    :param query: query in PG
+    :param LDAP: Flag for using LDAP credentials (defaults to False)
+    :param spatial: Flag for spatial table (defaults to True)
+    :param dest_schema: SQL Server schema for destination table (defaults to dest db's default schema)
+    :param dest_table: Table name of final migrated table in SQL Server database
+    :param print_cmd: Option to print he ogr2ogr command line statement (defaults to False) - used for debugging
+    :param temp: Flag for temporary table (defaults to True)
+    :return:
+    """
+    if not dest_schema:
+        dest_schema = ms.default_schema
+
+    if not dest_table:
+        dest_table = '_{u}_{d}'.format(u=ms.user, d=datetime.datetime.now().strftime('%Y%m%d%H%M'))
+
+    if spatial:
+        spatial = ' -a_srs EPSG:2263 '
+        nlt_spatial = ' '
+    else:
+        spatial = ' '
+        nlt_spatial = '-nlt NONE'
+
+
+    # apply regex to the query to filter out any dashed comments in the query
+    # comments are defined by at least 2 dashes followed by a line break or the end of the query
+    # comments with /* */ do not need to be filtered out from the query
+    query = re.sub('(-){2,}.*(\n|$)', '', query)
+
+    if LDAP:
+        cmd = PG_TO_SQL_QRY_CMD.format(
+            ms_pass='',
+            ms_user='',
+            pg_pass=pg.password,
+            pg_user=pg.user,
+            ms_server=ms.server,
+            ms_db=ms.database,
+            pg_host=pg.server,
+            pg_port=pg.port,
+            pg_database=pg.database,
+            sql_select = query,
+            ms_schema=dest_schema,
+            spatial=spatial,
+            dest_name=dest_table,
+            nlt_spatial=nlt_spatial,
+            gdal_data=GDAL_DATA_LOC
+        )
+    else:
+        cmd = PG_TO_SQL_QRY_CMD.format(
+            ms_pass=ms.password,
+            ms_user=ms.user,
+            pg_pass=pg.password,
+            pg_user=pg.user,
+            ms_server=ms.server,
+            ms_db=ms.database,
+            pg_host=pg.server,
+            pg_port=pg.port,
+            pg_database=pg.database,
+            sql_select = query,
+            ms_schema=dest_schema,
+            spatial=spatial,
+            dest_name=dest_table,
+            nlt_spatial=nlt_spatial,
+            gdal_data=GDAL_DATA_LOC
+        )
+
+    if print_cmd:
+        print(print_cmd_string([ms.password, pg.password], cmd))
+
+    try:
+        ogr_response = subprocess.check_output(shlex.split(cmd.replace('\n', ' ')), stderr=subprocess.STDOUT)
+        print(ogr_response)
+    except subprocess.CalledProcessError as e:
+        print("Ogr2ogr Output:\n", e.output)
+        print('Ogr2ogr command failed.')
+        raise subprocess.CalledProcessError(cmd=print_cmd_string([ms.password, pg.password], cmd), returncode=1)
+
+    ms.tables_created.append(dest_schema + "." + dest_table)
```

Comment:
the new format for this should be `ms.tables_created.append((ms.server, ms.database, dest_schema, dest_table))`